### DISCUSSION TITLE
When will Bluesky do a public beta?

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1377] [General Discussion]
I hope it is soon because Twitter is about to die and be "reborn" as X.

**Comment by Cererean** [comment]
Are you asking about when it will stop being invite only? The team has said somewhere they don't expect to ever drop the invite requirement, though they may speed up the pace to the point it's pretty much a non issue. The intent is to get federation done so access to the Bluesky server isn't need for access to the network.

**Comment by Zero3K** [comment]
Why won't they drop the invite requirement?

**Comment by Zero3K** [comment]
Is it because of an issue with not having enough servers?

**Comment by Zero3K** [comment]
Or is it a moderation issue?

**Comment by Zero3K** [comment]
I am closing this since it is now public.
